src/views/manage/components/TextInput.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { Input } from '@datapunt/asc-ui';
+
+const TextInput = ({ name, label, ...otherProps }) => {
```

Comment:
Is this component needed? It doesn't add anything to the `Input` component from `asc-ui`.

Reply:
It ignores the label param for the moment and will be extended in a next version of the app